### PR TITLE
masonry: add docs for `TreeArena::get_id_path`

### PR DESCRIPTION
**Pull request by tomcur**
beats reading the implementation to know the order :-)

### CODE REVIEW

File: masonry/src/tree_arena.rs [hunk 1]
```diff
@@ -207,6 +207,10 @@ impl<Item> TreeArena<Item> {
         Some(node.arena_mut(self.parents_map[&id], &mut self.parents_map))
     }
 
+    /// Construct the path of items from the given item to the root of the tree.
```
Comment (jaredoconnell) [inline masonry/src/tree_arena.rs:210]: Nit-pick: I would use active voice with "Constructs the path...", rather than an imperative command.

Comment (tomcur) [reply masonry/src/tree_arena.rs:210]: Thanks! It looks like much of the codebase currently is imperative, e.g., all methods on `Widget` have imperative docs.

Comment (jaredoconnell) [reply masonry/src/tree_arena.rs:210]: If that is the case, we'll stick to it for consistency.

**Review by DJMcNab — APPROVED** [review]
Thanks! Getting a first pass of documentation in places is very important.

**Pull request merged**